src/language-css/parser-postcss.js

```diff
@@ -390,6 +385,60 @@ function parseNestedCSS(node, options) {
       node.value = parseValue(value);
     }
 
+    if (node.type === "css-atrule") {
+      // mixin
+      if (node.mixin) {
```

Comment:
`isLessParser && node.mixin`

Reply:
Can't do that, `--parser=css` also need this, since we use less parser to parse css